packages/atomic/src/components/facets-v1/atomic-numeric-facet/atomic-numeric-facet.tsx

```diff
@@ -189,6 +223,12 @@ export class AtomicNumericFacet
     );
   }
 
+  private get valuesToRender() {
```

Comment:
Hides non-selected values with 0 count, like the regular facet

Reply:
Something to keep in mind (keep it as is for now, but FYI):

It was a semi frequent ask in JSUI to be able to display values with 0 count.

Reply:
For the ranges it's doable with an option on our side, for the category & regular it's going to take additional work on the API I think 🤔  For now I've kept all facets with the same behaviour